death to bullet 3

pdurbin opened the pull request.

pdurbin committed:
death to bullet 3

vsoch commented:
The idea of doing no harm is essential to being for the greater good. I'm open to suggestions for an improvement, but "death to bullet 3" doesn't seem constructive. It waters the affirmation down and removes one of it's core ideas.

pdurbin committed:
remove trailing space

For more context on the idea of removing bullet 3: https://gitter.im/good-labs/community?at=5cd4cdd0e7f42160fa6193d3

pdurbin commented:
You're right. I'm closing this pull request in favor of a "plain language" suggestion I just made in pull request #22
